backend/src/__tests__/authentication.test.js

```diff
@@ -0,0 +1,96 @@
+const { startTestServer, launchAPIServer } = require('./__utils');
+const jwt = require('jsonwebtoken');
+const { toPromise } = require('apollo-link');
+const { db } = require('../');
+const gql = require('graphql-tag');
+
+const HELLO = gql`
+  {
+    hello
+  }
+`;
+
+const userMock = {
+  id: '3',
+  fullName: 'John Doe',
+  trelloId: 'TRELLO_ID',
+  currentProject: {
+    id: 1,
+  },
+};
+
+const headers = {
+  Authentication:
+    'Bearer ' + jwt.sign({ id: userMock.id, trelloId: userMock.trelloId }, 'JWT_SECRET'),
+};
```

Comment:
Can you mock the authentication middleware in order to not have those line in each tests